### PR TITLE
Adds a few new recipes created with plastic sheets

### PR DESCRIPTION
**Pull request by Expletive**
:cl:
add: You can now create water bottles  and wet floor signs  by using plastic sheets. 
add: You can create toy swords out of plastic sheets, a piece of cable, and a light bulb.
/:cl:

@FantasticFwoosh added the plastic sheet production recipe, so I've gone ahead and added a few recipes for it. The toy sword mentions being made of plastic in it's description, and water bottles and wet floor signs look like they're made of plastic already.

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
I approve of this, though i would have liked it if you atomized the water spray, since that is a sensitive object and could torpedo this PR being accepted because of balance as just per my opinion.

I see no objection to the rest of it.

**Comment by Expletive** [comment]
@FantasticFwoosh Spray? No, these aren't spray bottles. They're the water bottles @Bawhoppen made with #26709. I just added recipes to craft empty versions of those.

File: code/modules/crafting/recipes.dm [hunk 1]
```diff
@@ -337,6 +337,12 @@
 		/obj/item/weapon/restraints/handcuffs/cable = 1
 	)
 	category = CAT_MISC
+	
+/datum/crafting_recipe/toysword
+	name = "Toy Sword" 
+	reqs[] = list(/obj/item/weapon/light/bulb= 1, /obj/item/stack/cable_coil = 1, /obj/item/stack/sheet/plastic = 4)
```
Comment (coiax) [inline code/modules/crafting/recipes.dm]: So it takes 4/5 of a plastic flaps to make one toy sword?

Comment (Expletive) [inline code/modules/crafting/recipes.dm]: Yeah. You have to have the handle, and a bunch of telescoping tubes. But I could reduce it if you want me to.

Comment (coiax) [inline code/modules/crafting/recipes.dm]: I suppose this seems a little delicate to be made by hand. I figured it would be in the autolathe.

**Comment by FantasticFwoosh** [comment]
Oh wait you're right.

**Comment by Bawhoppen** [comment]
fwoosh do you even read

**Comment by FantasticFwoosh** [comment]
I think faster than my lazy eyes can follow.

**Pull request merged**